it/connectors/ThirdPartyDeveloperConnectorEncryptionSpec.scala

```diff
@@ -31,7 +31,7 @@ import uk.gov.hmrc.http.HeaderCarrier
 class ThirdPartyDeveloperConnectorEncryptionSpec extends BaseConnectorIntegrationSpec with GuiceOneAppPerSuite {
   private val stubConfig = Configuration(
     "Test.microservice.services.third-party-developer.port" -> stubPort,
-    "json.encryption.key" -> "abcdefghijklmnopqrstuv=="
+    "json.encryption.key" -> "czV2OHkvQj9FKEgrTWJQZVNoVm1ZcTN0Nnc5eiRDJkY="
```

Comment:
I had to change this key as it was complaining it wasn't valid Base 64. Maybe we have a new version of the library that is more strict now. I just copied from the application config the same key we're using when running the app locally.